subsys/net/l2/wifi/wifi_shell.c

```diff
@@ -563,6 +565,11 @@ static int __wifi_args_to_params(const struct shell *sh, size_t argc, char *argv
 				&params->bssid[2], &params->bssid[3],
 				&params->bssid[4], &params->bssid[5]);
 			break;
+		case 't':
+			if (iface_mode == WIFI_MODE_INFRA) {
+				params->timeout = atoi(optarg);
```

Comment:
Better would be to use `shell_strtol()` that does error checks. The other `atoi` calls in this file should be fixed at some point, in different PR of course.

Reply:
Done